### PR TITLE
Allow reload pages with shortcut keys

### LINKED ISSUE #351: Allow reload to refresh timeline
## Describe
Make page reloading to refresh the timeline with `Cmd + r` or `Ctrl + r` or other commands. 

## Why
In my case, when start my PC from sleep with Whalebird app opened, my timeline has remained at the time of the start sleeping. Thus I believe the refreshing timeline feature will release us from restarting the app.
What do you think?

### PR DESCRIPTION
**Pull request by akito19**
Close #351 

Add shortcut keys, `Cmd + r` and `Ctrl + r`, to reload pages in Whablebird.

### CODE REVIEW

**Comment by h3poteto** [comment]
I am wondering whether to merge this.
Because it calls page reload function of Electron, but I did not suppose to reload all pages. I think that it is enough to reload current timeline and restart streaming processes.

When the page reload is called, are the unread marks reset?

**Comment by akito19** [comment]
I think that only reloading current timeline would not allow us to reload other timelines, hashtags, public and private timeline, etc. Need it in near future?
However, if reloading current timeline can restart streaming processes for these timelines, I'll fix my commit.

 > When the page reload is called, are the unread marks reset?

I'll confirm it after undo my development env because it hasn't shown any content with multiple log-in of the same account...

**Comment by h3poteto** [comment]
> I think that only reloading current timeline would not allow us to reload other timelines, hashtags, public and private timeline, etc. Need it in near future?

Yes, that's right. But we don't need to reload some pages.
Because some timelines are not updated with background.

File: src/renderer/components/TimelineSpace/Contents/Home.vue [hunk 1]
```diff
@@ -70,6 +72,43 @@ export default {
...
+        this.$message({
+          message: 'Could not find account',
+          type: 'error'
+        })
+      })
+      await this.$store.dispatch('TimelineSpace/stopUserStreaming')
+      await this.$store.dispatch('TimelineSpace/stopLocalStreaming')
+
+      await this.$store.dispatch('TimelineSpace/Contents/Home/fetchTimeline', account)
+        .catch(() => {
```
Comment (akito19) [inline src/renderer/components/TimelineSpace/Contents/Home.vue]: I attempt to `catch` errors on only current timeline although, in the case of `initialize()`, catch multiple streamings errors at the same time.

**Comment by h3poteto** [comment]
Nice! This is exactly what I hoped.

**Pull request merged**